### PR TITLE
Port a simplified gesture faker from polymer-gestures 0.5

### PR DESCRIPTION
**Pull request by azakus**

### CODE REVIEW

File: fake.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
+/*
```
Comment (cdata) [inline fake.js]: We are putting `@license` in all of these.

**Comment by cdata** [comment]
This looks nice, thanks for porting it. Some general feedback:

 - As @notwaldorf pointed out, `Fake` is a very generic name. I would propose an alternative such as: `FakeInteraction`.
 - Looking forward, some documentation on the methods might be nice (although don't consider that a blocker to merge).
 - I was wondering: how do you feel about reimplementing some (or all) of `mock-interactions.js` in terms of `Fake`?

File: mock-interactions.js [hunk 2]
```diff
@@ -1,27 +1,104 @@
...
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+ */
+
 (function(global) {
   'use strict';
 
-  function focus (target) {
-    Polymer.Base.fire.call(target, 'focus');
+  var HAS_NEW_MOUSE = (function() {
```
Comment (notwaldorf) [inline mock-interactions.js]: :mouse:

File: mock-interactions.js [hunk 3]
```diff
@@ -31,7 +108,21 @@
     });
   }
 
-  function keyboardEventFor (type, keyCode) {
+  function track(target, dx, dy, steps) {
+    dx = dx | 0;
+    dy = dy | 0;
```
Comment (cdata) [inline mock-interactions.js]: Did you intend the bit-wise operation here?

**Comment by cdata** [comment]
This looks good to me. Fake is dead. Long live Fake! :+1:

**Comment by notwaldorf** [comment]
MockInteractions lives another day! (lgtm too)

**Comment by azakus** [comment]
Hooray. I'll make an issue to update with docs.

**Pull request merged**